yamale/command_line.py

```diff
@@ -20,7 +20,15 @@
 
 
 def _validate(schema_path, data_path, parser, strict):
-    schema = schemas.get(schema_path)
+    try:
+        if os.path.getsize(schema_path):
```

Comment:
How does this raise an exception if the size of the file is 0?

Reply:
`os.path.getsize` - https://docs.python.org/3/library/os.path.html#os.path.getsize

I don't think I get your question. You can find information in the docs. Anything with a `if` clause in Python would check for a non null value. Right? Did I answer your question?

Reply:
Your error handling is in an `except` block, but there isn't anything that will raise the exception to get to your error handling code. How did you test that you get the message `schema.yml is an empty file!`?

Reply:
Does this answer the question - https://repl.it/repls/LightcyanEveryCalculator?

Reply:
If you print out the exception message, you'll see that is a `No such file or directory` error. If the file existed and is empty, no exception would be raised.

Reply:
KO - e8ea91435d73e37f5fafa6efa2241a13d43fa244 should fix.